metaswitch/common/comm_monitor.py

```diff
@@ -52,16 +52,21 @@ def __init__(self, process, raise_identifier, clear_identifier):
 
     def set_alarm(self):
         self.alarmed = True
+        _log.warning("Raising alarm {}".format(self._raise_identifier))
```

Comment:
Yes, I think it's a good idea to have a warning-level log associated with alarms - this means that it's visible in logs by default, so we can see when the system was in a bad state when reviewing logs, or track down "my alarm didn't get raised" bugs.